clar2wasm/src/tools.rs

```diff
@@ -373,3 +377,16 @@ pub fn crosscheck_validate<V: Fn(Value)>(snippet: &str, validator: V) {
 fn test_evaluate_snippet() {
     assert_eq!(evaluate("(+ 1 2)"), Ok(Some(Value::Int(3))));
 }
+
+#[test]
+fn test_fuel_limit() {
+    let mut env = TestEnvironment::new(StacksEpochId::latest(), ClarityVersion::latest(), Some(0));
+
+    assert!(env.evaluate("(+ 1 2)").is_err());
+
+    let mut env = TestEnvironment::new(StacksEpochId::latest(), ClarityVersion::latest(), Some(90));
```

Comment:
How do we know about `90`?

Reply:
trial and error, setting the limits will require a similar process